article/_posts/2021-07-23-DFC.md

```diff
@@ -0,0 +1,168 @@
+
+The total loss is the weighted sum of the terms of the two stages:
+
+$$L = L_{p} + \lambda L_{c}$$
+
+where $$L_{p} $$ is the distance prediction loss,
+$$L_{c} = KL(P \mid\mid Q) = \sum_{i} \sum{j} p_{ij} \log \frac{p_{ij}}{q_{ij}}$$,
+and $$\lambda$$ is a weighting term.
+
+In the above equation,
+
+$$p_{ij} = (q_{ij}^{2} / \sum_{i} q_{ij}) / (\sum_{j'} q_{ij'}^{2} / \sum_{i} q_{ij}))$$
+
+where $$q_{ij}$$ the soft assignment label (that follows a Student's
+t-distribution), i.e. the probability of assigning fiber $$i$$ to cluster $$j$$,
+and is computed as:
+
+$$q_{ij} = (1 + \lVert z_{i} - \mu_{j} \rVert^{2} * (1 - D_{ij}))^{-1} / (\sum_{j'} (1 + \lVert  z_{i} - \mu'_{j}  \rVert^{2} * (1 - D_{ij} 0 ))^{-1})$$
+
+where $$z_{i}$$ is the embedding of fiber $$i$$; $$\mu_{j}$$ is the centroid
+of cluster $$j$$; and $$D_{ij}$$ is the Dice score between the set of
+(Freesurfer) regions of fiber $$i$$ and the set of Freesurfer regions of cluster
+$$j$$.
+
+During inference, a fiber $$i$$ is assigned to the cluster with the maximum
+$$q_{ij}$$.
+
+
+## Data
+
+200 subjects of the HCP are used (split as 100/50/50 for
+training/validation/testing).
+
+For each training subject, 10000 random streamlines were selected, giving a
+total size of 1M training streamlines. For validation and testing, whole-brain
+tractography fibers (around 500k) were used.
+
+Streamlines below 40 mm were discarded from the tractograms.
+
+## Evaluation
+
+Quality measures:
+
+- Davies-Bouldin (DB) index:
+
+$$DB = (1/n) \sum_{k=1}^{n} max_{x \neq y} \frac{\alpha_{i} + \alpha_{j}}{d(c_{i}, c_{j})}$$
+
+where $$n$$ is the number of clusters, $$\alpha_{i}$$ and $$\alpha_{j}$$ are 
+mean pairwise intra-cluster fiber distances, and $d(c_{i}, c_{j})$ is the
+inter-cluster fiber distance between centroids $$c_{i}$$ and $$c_{j}$$ of
+cluster $$i$$ and $$j$$. A smaller DB score indicates a better separation
+between clusters.
+
+- White Matter Parcellation Generalization (WMPG): represents the percentage of
+clusters successfully detected across the testing subjects.
+
+- Tract Anatomical Profile Coherence (TAPC): measures if the fibers within a
+cluster $$c$$ commonly pass through the same brain anatomical regions:
+
+$$TAPC(c) = (\sum_{f = 1}^{NF(c)} Dice(TAP(f), TAP_{atlas}(c)))/NF(c)$$
+
+Higher TAPC scores indicate better anatomical coherence.
+
+Baselines are QuickBundles[^1] and WMA[^2].
+
+
+# Results
+
+Quality measures:
+
+![](/article/images/DFC/Results_table.jpg)
+
+Fiber clusters:
+
+![](/article/images/DFC/Results_plot.jpg)
+
+Outlier removal:
+
+![](/article/images/DFC/Outlier_removal.jpg)
+
+
+# Conclusions
+
+Authors presented a novel unsupervised deep learning framework for dMRI
+tractography clustering.
+
+The proposed method shows improved performance over the compared baselines.
+
+
+# References
+
+[^1]: Garyfallidis, E., Brett, M., Correia, M.M., et al. Quickbundles, a method
+for tractography simplification. Frontiers in neuroscience 6, 175 (2012)
```

Comment:
Seems like you should avoid line breaks. Currently, the lines that don't start with `[^` are all grouped right below the section heading.

Reply:
Thanks for noticing. 4a223c8 should fix the issue: using the appropriate indentation should no longer pose a problem. It has been used in other posts (e.g. https://github.com/vitalab/vitalab.github.io/blob/master/article/_posts/2019-01-31-InstantBrainAtlasSegmentation.md).